src/chrome/content/rules/Uni.edu.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="UNI">
+        <target host="uni.edu" />
+        <target host="*.uni.edu" />
```

Comment:
This breaks `scholarworks.uni.edu`.
In general, we try to avoid wildcards. If there is a good reason for a wildcard, you should add a bunch of test urls for it.